src/app/inventory/store/d1-store-factory.ts

```diff
@@ -100,23 +58,23 @@ export function makeCharacter(
   const race = defs.Race[character.characterBase.raceHash];
   let genderRace = '';
   let className = '';
-  let gender = '';
-  let genderName = '';
+  let gender: DimStore['gender'] = '';
+  let genderName: DimStore['genderName'] = '';
   if (character.characterBase.genderType === 0) {
     gender = 'male';
-    genderName = gender;
+    genderName = 'male';
```

Comment:
is this a d1 vs d2 thing?

Reply:
also wonder if there is a static value we could pull from DestinyGender, etc

Reply:
Yeah this is just D1. There may be a definition, but we weren't using them all back then.